apps/data_handler/handlers/loan_states/nostra_alpha/events.py

```diff
@@ -568,12 +571,15 @@ def process_interest_bearing_collateral_burn_event(self, event: pd.Series) -> No
         # The order of the values in the `data` column is: `user`, `amount`, ``.
         # Example:
         # https://starkscan.co/event/0x0106494005bbab6f01e7779760891eb9ae20e01b905afdb16111f7cf3a28a53e_1.
-        user = event["data"][0]
+         
+        data = NostraDataParser.parse_interest_bearing_collateral_mint_event(event["data"])
+        user, amount = data.user, data.amount
```

Comment:
please, be specific which amount, we have `face` and `raw` and here it's very important